packages/appstore/webpack.config.js

```diff
@@ -108,7 +108,7 @@ module.exports = function (env) {
                         {
                             loader: 'css-loader',
                             options: {
-                                url: false,
+                                url: (_, resourcePath) => resourcePath.includes('packages/wallets'),
```

Comment:
Set url resolution for CSS to `true` only when bundling up wallets package, this ensures that the change only affects wallets package

Reply:
This is required otherwise the `url` in the css will not be resolved to the path in the bundle